### PR TITLE
Add NeoVim cleaner

### PR DESCRIPTION
**Pull request by dngray**

### CODE REVIEW

File: pending/nvim.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+-->
+
+<cleaner id="nvim">
```
Comment (az0) [inline pending/nvim.xml]: Please add `os="linux"`

File: pending/nvim.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  <label>Neovim</label>
+  <description>Editor</description>
+  <option id="history">
+    <label>History</label>
+    <description>Delete netrw directory history, command history</description>
+    <action search="file" command="delete" path="~/.config/nvim/.netrwhist"/>
+    <action search="file" command="delete" path="~/.local/share/nvim/shada/main.shada"/>
```
Comment (az0) [inline pending/nvim.xml]: The best practice is to use the XDG_ variables as described in the [documentation on variables](https://docs.bleachbit.org/cml/variables.html)

File: pending/nvim.xml (hunk 1 above)
Comment (dngray) [reply pending/nvim.xml]: Done, thanks for the advice.

File: pending/nvim.xml (hunk 2 above)
Comment (dngray) [reply pending/nvim.xml]: Done, thanks for the advice.

**Comment by dngray** [comment]
Done.

**Pull request merged**